sqlalchemy_ibmi/base.py

```diff
@@ -918,16 +939,19 @@ def _get_driver_version(self, db_conn):
     ischema = MetaData()
 
     sys_schemas = Table(
-        "SQLSCHEMAS", ischema,
-        Column("TABLE_SCHEM", sa_types.Unicode, key="schemaname"),
-        schema="SYSIBM")
+        'sysschemas',
+        ischema,
+        Column('schema_name', sa_types.Unicode, key="schemaname"),
+        schema='QSYS2'
```

Comment:
```suggestion
        'SYSSCHEMAS',
        ischema,
        Column('SCHEMA_NAME', sa_types.Unicode, key="schemaname"),
        schema='QSYS2'
```

For consistency with other metadata table definitions.